src/components/ProjectSearch.js

```diff
@@ -147,12 +147,12 @@ export class ProjectSearch extends Component<Props, State> {
 
   selectMatchItem = (matchItem: Match) => {
     this.props.selectSpecificLocation({ ...matchItem });
-    this.props.doSearchForHighlight(
-      this.state.inputValue,
-      getEditor(),
-      matchItem.line,
-      matchItem.column
-    );
+    // this.props.doSearchForHighlight(
+    //   this.state.inputValue,
+    //   getEditor(),
+    //   matchItem.line,
+    //   matchItem.column
+    // );
```

Comment:
why isn't this needed?

Reply:
Hi @jasonLaster 
We still need it, I just changed the place to 
https://github.com/rebornix/debugger.html/pull/2/files#diff-cfc118adfafc73d553d9d93a28eb0a15R417